src/microbe_stage/biomes.cpp

```diff
@@ -161,7 +161,7 @@ CScriptArray*
         Leviathan::ScriptExecutor::Get()->GetASEngine(), "array<uint64>");
 }
 
-// ----- //
+// ------------------------------------ //
```

Comment:
not sure how necessary this comment change  is.

Reply:
It is 100% necessary and unlikely to cause a merge conflict.

Reply:
Ahh, looking at the actual file i see that was the one /---- that wast nthe same size as the rest. I feel your pain.